how-to/networking/dnssec-troubleshooting.md

```diff
@@ -0,0 +1,260 @@
+(dnssec-troubleshooting)=
+# Basic DNSSEC troubleshooting
+
+Some of the troubleshooting tips that will be shown here are focused on the bind9 DNS server and its tools, but the general principle applies to DNSSEC in all implementations.
+
+## Handy "bad" and "good" DNSSEC domains
+It helps to have some good known domains with broken and working DNSSEC available for testing, so we can be sure our tooling is catching those, and not just failing everywhere. There is no guarantee that these domains will be up forever, and certainly there are more out there, but this list is a good first choice:
+
+ * These should fail DNSSEC validation:
+   * *dnssec-failed.org*
+   * *sigfail.ippacket.stream*
+ * These should pass DNSSEC validation:
+   * *isc.org*
+   * *sigok.ippacket.stream*
+
+## Logs
+
+By default, the bind9 server will log certain DNSSEC failures, and the journal log should be the first place to check.
+
+For example, if we ask a bind9 Validating Resolver for the IP address of the *www.dnssec-failed.org* name, we get a failure:
```

Comment:
```suggestion
For example, if we ask a bind9 Validating Resolver for the IP address of the www.dnssec-failed.org name, we get a failure:
```

Reply:
I ended up using back ticks, as this is not meant to be clicked upon.